notebooks/Custom_Model_TFX.ipynb

```diff
@@ -1,1339 +1,1839 @@
 {
```

Comment:
<p>If some imports are unused then discard them. Also format the component modules with <code>black</code>  which is what I also did. This way the linings are beautifully formatted. I'd also log the TFLite model URI with <code>logging.info()</code>.  There's a small typo in: <code>logging.info(f"(MY)doanload model.tflite is completed")</code>.</p><p> <span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sayakpaul/Dual-Deployments-on-Vertex-AI/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>